api/src/app.ts

```diff
@@ -99,12 +109,17 @@ export const build = async (
   await fastify.register(cookies);
   await fastify.register(csrf);
 
+  await fastify.register(growthBook, {
+    apiHost: GROWTHBOOK_FASTIFY_API_HOST,
+    clientKey: GROWTHBOOK_FASTIFY_CLIENT_KEY
+  });
+
   const provider =
     EMAIL_PROVIDER === 'ses' ? new SESProvider() : new NodemailerProvider();
   void fastify.register(mailer, { provider });
 
   // Swagger plugin
-  if (FCC_ENABLE_SWAGGER_UI) {
+  if (FCC_ENABLE_SWAGGER_UI || fastify.gb.isOn('swagger-ui')) {
```

Comment:
One issue with this is that we can't turn them off locally if they're on in GB.

We could work around this by leaving `FCC_ENABLE_SWAGGER_UI` as `undefined | string` or `undefined | boolean` (i.e. allow for it to be on, off or not set). That does complicate things a bit, though.

Reply:
True, but not having the growthbook uri prevents that.